code-of-conduct.md

```diff
@@ -1,120 +1,204 @@
-    *   Unwelcome sexual attention.
-    *   Advocating for, or encouraging, any of the above behavior.
-    *   Repeated harassment of others. In general, if someone asks you to stop,
-        then stop.
-
-*   **When we disagree, try to understand why**: Disagreements, both social and
-    technical, happen all the time. It is important that we resolve
-    disagreements and differing views constructively. Remember that we’re
-    different. The strength of our community comes from its diversity, people
-    from a wide range of backgrounds. Different people have different
-    perspectives on issues. Being unable to understand why someone holds a
-    viewpoint doesn’t mean that they’re wrong. Don’t forget that it is human to
-    err and blaming each other doesn’t get us anywhere. Instead, focus on
-    helping to resolve issues and learning from mistakes.
+# Kinetic Commerce Open Source Code of Conduct
+
+Version 2.0
+
+This code of conduct outlines our expectations for participating in Kinetic
+Commerce [open source projects][kineticcafe], as well as steps to reporting
+unacceptable behaviour.
+
+We are committed to providing a welcoming and inspiring community for all and
+expect our code of conduct to be honored. Anyone who violates this code of
+conduct may be banned from participation on one or more Kinetic Commerce open
+source projects.
+
+We strive to:
+
+- **Be friendly and patient.**
+
+- **Be welcoming**: We strive to make participation in our projects
+  a harassment-free experience for everyone, regardless of age, body size,
+  caste, color, culture, education, ethnicity, family status, gender identity
+  and expression, language, immigration status, level of experience, national
+  origin, personal appearance, political beliefs, profession, race, religion,
+  sex characteristics, sexual identity and orientation, socio-economic status,
+  or visible or invisible disability.
+
+- **Be considerate**: Your work will be used by other people, and you in turn
+  will depend on the work of others. Any decision you take will affect users
+  and colleagues, and you should take those consequences into account when
+  making decisions. Remember that we're a world-wide community, so you might
+  not be communicating in someone else's primary language.
+
+- **Be respectful**: Not all of us will agree all the time, but disagreement
+  is no excuse for poor behavior or poor manners. We all experience some
+  frustration now and then, but we cannot allow that to turn into personal
+  attacks. It's important to remember that a community where people feel
+  uncomfortable or threatened is not a productive one.
+
+- **Be careful in the words that you choose**: we are a community of
+  professionals, and we conduct ourselves professionally. Be kind to others.
+  Do not insult or put down other participants. Harassment and other
+  exclusionary behavior aren't acceptable.
+
+  Behaviour that that contribute to a positive environment include:
```

Comment:
- `that` is repeated
- My preference is for “Behaviour that contributes…” or “Behaviours that contribute…”